sensor_msgs/PointCloud2 cloud
geometry_msgs/PoseStamped pose
---
sensor_msgs/PointCloud2 cloud
